lib/2fa.coffee

```diff
@@ -47,7 +47,7 @@ get2fa = (deps, opts) ->
 	# });
 	###
 	exports.isEnabled = (callback) ->
-		token.getProperty('twoFactorRequired').then (twoFactorRequired) ->
+		auth.getProperty('twoFactorRequired').then (twoFactorRequired) ->
```

Comment:
Resin auth doesn't look like it has a `getProperty` method, so it seems unlikely that this works.

I think (@flesler can confirm?) API keys never need 2FA, so I think these methods should have `enabled` return false and `passed` return true, if you're using an api key.

Reply:
Indeed, it doesn't have it (we need a test then!) and API Keys don't have a test, so it's always `true`